src/test/codegen/sanitizer-cfi-emit-type-metadata-id-itanium-cxx-abi.rs

```diff
@@ -425,6 +429,8 @@ pub fn foo145(_: Type13, _: Type13) { }
 // CHECK: define{{.*}}foo145{{.*}}!type ![[TYPE145:[0-9]+]]
 pub fn foo146(_: Type13, _: Type13, _: Type13) { }
 // CHECK: define{{.*}}foo146{{.*}}!type ![[TYPE146:[0-9]+]]
+pub fn foo147(_: Type14<u8>) { }
```

Comment:
Would you mind adding functions with two and three parameters (i.e., foo148 and foo149) similarly to other types to test compression/substitution works for this type?

Reply:
Done.